### DISCUSSION TITLE
An error occurs when using it in Vue.

### DISCUSSION THREAD

**Discussion by fPdoinb** [markmap/markmap#200] [Q&A]
After referencing the documentation, when using it in Vue, the following error is thrown: Uncaught (in promise) TypeError: Cannot read properties of undefined (reading 'createElementNS') at Proxy. (creator.js:10:20) at Proxy. (append.js:6:36) at Selection.select_default [as select] (select.js:9:50) at Selection.append_default [as append] (append.js:5:15) at new _Markmap (index.js:878:31) at _Markmap.create (index.js:1230:16) at Photo.vue:179:18 at runtime-core.esm-bundler.js:2675:88 at callWithErrorHandling (runtime-core.esm-bundler.js:158:18) at callWithAsyncErrorHandling (runtime-core.esm-bundler.js:166:16). Please provide a solution.

**Comment by gera2ld** [comment]
Please disable SSR for at least the component.
